66.py

```diff
@@ -0,0 +1,18 @@
+class Solution:
+    def plusOne(self, digits: List[int]) -> List[int]:
+        carry = 1
+
+        for i in range(len(digits) - 1, -1, -1):
+            digit = digits[i] + carry
+
+            if digit > 9:
+                digits[i] = 0
+                carry = 1
+            else:
+                digits[i] = digit
+                carry = 0
```

Comment:
You can enable early return here. If there is no carry at this digit, there won't be any carries afterwards.